dna/models/baselines.py

```diff
@@ -6,8 +6,11 @@
 import pandas as pd
 from sklearn import linear_model
 from sklearn.ensemble import RandomForestRegressor
+import torch
+from .torch_modules.mlp import MLP
 
-from .base_models import RankModelBase, RegressionModelBase, SklearnBase, SubsetModelBase
+from .base_models import RankModelBase, RegressionModelBase, SklearnBase, SubsetModelBase,\
```

Comment:
A common practice for multi-line imports is to is parentheses:
```
from module import (
    a, b, c, d, e
    f, g, h, i, j
)
```
It's not that important though.

Reply:
Thanks for letting me know.